content/content-strategy/audit-content/intro.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: intros/intro_with_nav
+subtitle: Audit your content to get a clear understanding of your digital estate or service. Create evidence to improve, remove or change.
+attrTitle1: Created by
+attrValue1: Digital Transformation Agency and Department of Industry, Innovation and Science
+attrTitle2: Reviewed on 1 March 2018
+attrValue2: 
```

Comment:
`attrValue1:` should be just Reviewed on' This is the label part of the `<dl>` (Description List) element
`attrValue2:` should be '1 March 2018'

Reply:
Trisha - looked ok